.rubocop.yml

```diff
@@ -0,0 +1,26 @@
+require:
+  - rubocop-rails
+  - rubocop-rspec
+
+AllCops:
+  NewCops: enable
+  Exclude:
+    - 'db/**/*'
+    - 'config/**/*'
+    - 'bin/**/*'
+    - 'node_modules/**/*'
+    - 'tmp/**/*'
+    - 'vendor/**/*'
+    - '.git/**/*'
+    - 'Gemfile'
+    - 'Rakefile'
+    - 'Gemfile.lock'
+    - 'config.ru'
+
+Style/Documentation:
+  Enabled: false
+Style/FrozenStringLiteralComment:
+  Enabled: false
```

Comment:
what's the point of disabling this rule?
in case it was made because existing files are missing this top level comment, it worth to run `rubocop -a` in order to fix it without disabling rule at all